cvat-core/src/annotations-objects.ts

```diff
@@ -2996,11 +2996,13 @@ export class SkeletonTrack extends Track {
         if (updatedOccluded.length) {
             updatedOccluded.forEach((el) => { el.updateFlags.occluded = true; });
             updateElements(updatedOccluded, HistoryActions.CHANGED_OCCLUDED);
+            this.saveOccluded(data.occluded, frame);
```

Comment:
In skeletons properties like ``occluded``, ``outside``, ``hidden``, ``lock`` are computable.

For example occluded:
```
get occluded(): boolean {
        return this.elements.every((element) => element.occluded);
    }
```

This is not necessary to set them. Also if one element of skeleton is occluded, it does not make the whole skeleton occluded.

Reply:
Thanks. I fixed this issue on the server side.